packages/ado-extension/task.json

```
@@ -29,61 +29,47 @@
         {
             "name": "hostingMode",
             "type": "pickList",
-            "label": "Hosting Mode",
+            "label": "Hosting mode",
             "required": true,
             "options": {
-                "staticSite": "Static Site (this task runs a localhost webserver for your site)",
-                "dynamicSite": "Dynamic Site (you host your site separately)"
+                "staticSite": "Static site (HTML files in a local directory)",
+                "dynamicSite": "URL (localhost, staging URL, or production URL)"
             },
-            "helpMarkDown": "Note: not required in YAML configuration, only on classic pipelines UI. Your site must be served (hosted) before it can be scanned. In _Static Site_ mode, this task will run a localhost webserver serving your site directory and scan that. In _Dynamic Site_ mode, you must host your site yourself separately and specify a URL to scan. This can be either a localhost server you run in an earlier pipeline step, or a remote URL (for example, a staging environment)."
+            "helpMarkDown": "Note: not required in YAML configuration, only on classic pipelines UI. Your site must be served (hosted) before it can be scanned. In _Static site_ mode, this task will run a localhost webserver serving your site directory and scan that. In _URL_ mode, you must host your site yourself separately and specify a URL to scan. This can be either a localhost server you run in an earlier pipeline step, or a remote URL (for example, a staging environment).\n\nWhen switching between hosting modes in the classic pipeline, remove values for the previous hosting mode to prevent caching."
         },
         {
             "name": "staticSiteDir",
             "type": "string",
-            "label": "Static Site Directory",
+            "label": "Hosting mode: site directory",
             "required": true,
-            "helpMarkDown": "Folder containing website content.",
+            "helpMarkDown": "Directory containing the website content to be scanned.",
```

Would it be worth hammering home that the website content needs to be HTML files?
```suggestion
            "helpMarkDown": "Directory containing the HTML files to be scanned.",
```